### PR TITLE
feat: add combo perk

### PR DESCRIPTION
**Pull request by LordMidas**

### CODE REVIEW

File: scripts/skills/perks/perk_rf_combo.nut [hunk 1]
```diff
@@ -0,0 +1,81 @@
+this.perk_rf_combo <- ::inherit("scripts/skills/skill", {
+	m = {
+		UsedSkillID = "",
+		UsedSkillName = ""
+	},
+	function create()
+	{
+		this.m.ID = "perk.rf_combo";
+		this.m.Name = ::Const.Strings.PerkName.RF_Combo;
+		this.m.Description = ::Const.Strings.PerkDescription.RF_Combo;
```
Comment (Darxo) [inline scripts/skills/perks/perk_rf_combo.nut:10]: The perk description is not suited to be the header description of this perk in effect form when viewing the tooltip

File: scripts/skills/perks/perk_rf_combo.nut [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	function onWaitTurn()
+	{
+		this.m.UsedSkillID = "";
+	}
+
+	function onTurnStart()
+	{
+		this.m.UsedSkillID = "";
```
Comment (Darxo) [inline scripts/skills/perks/perk_rf_combo.nut:72]: You do not reset `UsedSkillID` onTurnEnd (as in, this skill is not deactivated then)
That will cause confusion:
- you wonder how long this effect is active after it has not vanished from a character.
- you might be confused by the AP cost of enemy attacks when hovering over them after they ended their turn.

File: scripts/skills/perks/perk_rf_combo.nut [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+	}
+
+	function onCombatFinished()
+	{
+		this.skill.onCombatFinished();
+		this.m.UsedSkillID = "";
+		this.m.UsedSkillName = "";
```
Comment (Darxo) [inline scripts/skills/perks/perk_rf_combo.nut:79]: It is redundant to reset this variable.

**Review by Darxo — CHANGES_REQUESTED** [review]

**Review by Darxo — APPROVED** [review]

**Comment by LordMidas** [comment]
I just realized that one thing missing from this perk is the affordability preview modification. And this is one of those cases where my [improved affordability preview](https://github.com/MSUTeam/MSU/pull/346) implementation over at MSU will be required because we need to check for the `ActionPointCost` of those skills to be above 3 before this skill modifies them.

**Review by Darxo — APPROVED** [review]

**Review by Darxo — APPROVED** [review]

**Pull request merged**